plot_pixel_values.py

```diff
@@ -22,41 +24,103 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
```

Comment:
Ah,  This was an option to plot the Pixel Values vs T_CCD in a separate figure.  I think it needs a new name.